titty

```diff
@@ -44,10 +44,10 @@ tdisp(){
 	total=0
 	while read -r NAME START STOP
 	do
-		if [ "$1" = 'all' ] || [ "$(date --date=@"$START" +"$FMTSTRING")" -eq "$(date +"$FMTSTRING")" ]
```

Comment:
Here if the format string were to have a space then date will take it as 2 different arguments causing it trouble